services/meeting_service.js

```diff
@@ -0,0 +1,90 @@
+const {models} = require('../db/relations')
+const {Checker} = require('./data_checker')
+const Meeting = models.Meeting
+const Guest = models.Guest
+const Host = models.Host
+
+const findOrCreateGuest = async(email) => {
+    const guest = await Guest.findOrCreate({where:{email: email}})
+    return guest[0];
+}
+
+const findOrCreateHosts = async(emails) => {
+    const hostsArray = []
+    for(const email of emails) hostsArray.push(await Host.findOrCreate({where: {email: email}}))
+    return hostsArray.map( hostArray => hostArray[0])
+}
+
+const isNewHosts = async (meeting,hostsMails) => {
+    const meetingHosts = await meeting.getHosts();
+    const meetingMails = meetingHosts.map(host => host.emial)
+    return JSON.stringify(meetingMails) === JSON.stringify(hostsMails)
+}
+
+const isNewGuest = async (meeting,guestMail) => {
+    const guest = await meeting.getGuest();
+    return guest.email === guestMail
+}
+
+const updateMeetingHosts = async(meeting,hostsMails) => {
+    if(await isNewHosts(meeting,hostsMails))return;
+    const hosts = await findOrCreateHosts(hostsMails)
+    await meeting.setHosts(hosts)
+}
+
+const updateMeetingGuest = async (meeting,guestMail) => {
+    if(await isNewGuest(meeting,guestMail))return;
+    const guest = await findOrCreateGuest(guestMail)
+    await meeting.setGuest(guest)
+}
+
+const updateMeetingDuration = async(meeting, duration) => 
+    {if(meeting.duration!==duration) await meeting.update({duration: duration}, {where: {uuid: uuid}})}
+
+const createResponse = (msg, status = 200) => {return {
```

Comment:
to pewnie wyciągniemy do jakichś commonsów, ale to może po mergu ja swój kod dostosuje